deps/GraphBLAS/CMakeLists.txt

```diff
@@ -60,17 +60,18 @@ include_directories ( Source/Template Source Source/Generated Include Demo/Inclu
 if ("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")
     # cmake 2.8 workaround: gcc needs to be told to do ANSI C11.
     # cmake 3.0 doesn't have this problem.
-    set (CMAKE_C_FLAGS  "-std=c11 -lm -fPIC")
+    set (CMAKE_C_FLAGS  "-g -std=c11 -lm -fopenmp -fPIC")
     if (CMAKE_C_COMPILER_VERSION VERSION_LESS 4.9)
         message (FATAL_ERROR "gcc version must be at least 4.9")
     endif ( )
 elseif ("${CMAKE_C_COMPILER_ID}" STREQUAL "Intel")
     # options for icc: also needs -std=c11
-    set (CMAKE_C_FLAGS  "-std=c11 -fopenmp")
+    set (CMAKE_C_FLAGS  "-g -std=c11 -fopenmp")
     if (CMAKE_C_COMPILER_VERSION VERSION_LESS 18.0)
         message (FATAL_ERROR "icc version must be at least 18.0")
     endif ( )
 elseif ("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
+    set (CMAKE_C_FLAGS  "-g -fopenmp")
```

Comment:
@swilly22 Strangely, we hadn't been adding any flags to Clang compilation before (nor are there any in this makefile in stock GraphBLAS). We don't even need -fPIC, though? I'm not sure how that worked/works.